src/postprocess.ts

```diff
@@ -157,12 +156,12 @@ export class PostProcess {
       this.document.setKeywords(keywords);
     }
 
-    const creator = tree[metaTerms.contributor]?.find(
-      (item) => item.r?.[metaTerms.role]?.[0].v === 'bkp',
-    )?.v;
```

Comment:
I wonder about this removed code. Probably, "bkp" means "Book producer" (I find it in the [MARC Code List for Relators](https://www.loc.gov/marc/relators/relacode.html)), and this metadata is not for the CreatorTool that is set by `this.document.setCreator()` here. So I think removing this code is probably ok.

@zwaldowski Please check this.